src/pages/Trace/Instructions/Instructions.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const button: Button = {text: "Get Started", to: "/AddPhoto" };
```

Comment:
this button seems unnecessary

Reply:
The button will be used when we add the done button to this page. I added a todo at first, but I think its better to remove it for now.